src/app/models/table-models/dimension.ts

```diff
@@ -0,0 +1,25 @@
+export class Dimension {
```

Comment:
I thought of a bit different model for dimension - to include index, label, maybe elements as well (to match Jan's response format a bit). I'm not sure what does selected mean?

Reply:
yes, you can add extra fields to Dimension of course

Reply:
'selected' is a flag to indicate if the dimension is selected or not